ui/src/shared/components/LayoutCellMenu.tsx

```diff
@@ -11,6 +11,11 @@ import {Cell} from 'src/types/dashboards'
 import {QueryConfig} from 'src/types/queries'
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
+enum VisType {
```

Comment:
Since you have this in 2 places, maybe move to an outside file and import into both?

Reply:
will do!